tools/check-commit-message.sh

```diff
@@ -4,19 +4,19 @@ set -o nounset
 
 commit_file=${1}
 commit_message="$(cat ${commit_file})"
-valid_commit_regex='([A-Z]+-[0-9]+|#[0-9]+|merge|no-issue|revert)'
+valid_commit_regex='^(Bug [0-9]+|[A-Z]+-[0-9]+|#[0-9]+|Merge|NO-ISSUE|Revert)'
```

Comment:
I think we need to have ":" after the bug number
@flaper87 ?

Reply:
I think it's optional but we let's require it here just for good measure.

/cc @celebdor

Reply:
Done